docs/docsite/rst/inventory_guide_iocage_aliases.rst

```diff
@@ -0,0 +1,208 @@
+   Please convert back to a jail before trying to start ansible_client
+
+List the jails:
+
+.. code-block:: console
+
+   shell> iocage list -l
+   +-----+----------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+   | JID |   NAME   | BOOT | STATE | TYPE |     RELEASE     |        IP4         | IP6 |    TEMPLATE    | BASEJAIL |
+   +=====+==========+======+=======+======+=================+====================+=====+================+==========+
+   | 207 | 052b9557 | off  | up    | jail | 14.2-RELEASE-p3 | epair0b|10.1.0.137 | -   | ansible_client | no       |
+   +-----+----------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+   | 208 | 1c11de2d | off  | up    | jail | 14.2-RELEASE-p3 | epair0b|10.1.0.146 | -   | ansible_client | no       |
+   +-----+----------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+   | 209 | 9d94cc9e | off  | up    | jail | 14.2-RELEASE-p3 | epair0b|10.1.0.115 | -   | ansible_client | no       |
+   +-----+----------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+
+Set notes. The tag *alias* will be used to create inventory aliases:
+
+.. code-block:: console
+
+   shell> iocage set notes="vmm=iocage_02 project=foo alias=srv_1" 052b9557
+   notes: none -> vmm=iocage_02 project=foo alias=srv_1
+   shell> iocage set notes="vmm=iocage_02 project=foo alias=srv_2" 1c11de2d
+   notes: none -> vmm=iocage_02 project=foo alias=srv_2
+   shell> iocage set notes="vmm=iocage_02 project=bar alias=srv_3" 9d94cc9e
+   notes: none -> vmm=iocage_02 project=bar alias=srv_3
+
+Update the inventory configuration. Set the option
+:ansopt:`community.general.iocage#inventory:inventory_hostname_tag` to *alias*. This tag keeps the
+value of the alias. The option :ansopt:`community.general.iocage#inventory:get_properties` must be
+enabled:
+
+.. code-block:: console
+
+   shell> cat hosts/02_iocage.yml
+
+.. code-block:: yaml
+
+   plugin: community.general.iocage
+   host: 10.1.0.73
+   user: admin
+   get_properties: true
+   inventory_hostname_tag: alias
+   hooks_results:
+     - /var/db/dhclient-hook.address.epair0b
+   compose:
+     ansible_host: (iocage_hooks.0 == '-') | ternary(iocage_ip4, iocage_hooks.0)
+     iocage_tags: dict(iocage_properties.notes | split | map('split', '='))
+   keyed_groups:
+     - prefix: vmm
+       key: iocage_tags.vmm
+     - prefix: project
+       key: iocage_tags.project
+
+Display tags and groups. Create a playbook:
+
+.. code-block:: console
+
+   shell> cat pb-test-groups.yml
+
+.. code-block:: yaml+jinja
+
+   - hosts: all
+     remote_user: admin
+
+     vars:
+
+       ansible_python_interpreter: auto_silent
+
+     tasks:
+
+       - debug:
+	   var: iocage_tags
+
+       - debug:
+           msg: |
+	   {% for group in groups %}
+	   {{ group }}: {{ groups[group] }}
+	   {% endfor %}
```

Comment:
Note that there are tab characters in this file (for example the start of the above lines). Please make sure you don't use any tab characters in RST files.

Also please note that the message is not indented correctly - though that might be caused by the tabs.

Reply:
Done.